src/net/sf/memoranda/ui/WorkPanel.java

```diff
@@ -259,9 +273,21 @@ public void eventsB_actionPerformed(ActionEvent e) {
 	}
 
 	public void filesB_actionPerformed(ActionEvent e) {
-		cardLayout1.show(panel, "FILES");
-		setCurrentButton(filesB);
-		Context.put("CURRENT_PANEL", "FILES");
+		if (Configuration.get("RESOURCE_PATH").equals("nothing")) {
```

Comment:
existing users will already have a .preferences file in their user directory, so they will not get the RESOURCE_PATH item from the default file unless their preferences file is deleted, which is not good UX